### ISSUE TITLE
Message editing: attach message as reply to other message via Edit form

### ISSUE THREAD

**Issue by MurzNN** [vector-im/element-meta#1047]

**Comment by t3chguy** [comment]
@MadLittleMods the MSC says

> Any m.relates_to property within m.new_content is ignored.

Which implies you cannot attach a relation via an edit.

https://github.com/matrix-org/matrix-spec-proposals/blob/main/proposals/2676-message-editing.md#applying-mnew_content

**Comment by MadLittleMods** [comment]
@t3chguy 👍 The spec is against this usage but it can technically work.

**Comment by t3chguy** [comment]
> @t3chguy 👍 The spec is against this usage but it can technically work.

In in a faulty client, does synapse include the new relation in unsigned relations? Do other clients accept it too?

**Comment by MadLittleMods** [comment]
@t3chguy Yes, it works in Element after https://github.com/matrix-org/matrix-react-sdk/pull/6817

This is from before it became much more clear/crystalized in the MSC's (at least to me) that this was against current spec.

**Comment by t3chguy** [comment]
I suggest opening an MSC or clarification because that pr will need reverting based on spec compliance otherwise

**Comment by MadLittleMods** [comment]
@t3chguy I've created https://github.com/matrix-org/matrix-spec/issues/1452 to track this. I think we should also keep this issue open to track the feature itself and how that would look like vs whether the spec actually allows for it at the moment.

**Comment by t3chguy** [comment]
Agreed, this issue is about the feature regardless of underlying implementation
